lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -535,6 +540,42 @@ def _delete_environment(self, environment: Environment, log: Logger) -> None:
             else:
                 log.debug("not wait deleting")
 
+    def save_console_log(
```

Comment:
It copies too many code. Try to use construct the nodes earlier and use serial console feature.

Reply:
feature was not initialized at that time; I moved the common login into method into common.py file. Let know any concern.